### ISSUE TITLE
This is how to fix the lag

### ISSUE THREAD

**Issue by codydixon** [drelleum/OldSchoolRS#7]
cd
wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/7u60-b19/jdk-7u60-linux-arm-vfp-hflt.tar.gz
tar -C /opt -zxvf jdk-7u60-linux-arm-vfp-hflt.tar.gz
rm jdk-7u60-linux-arm-vfp-hflt.tar.gz
update-alternatives --install /usr/bin/java java /opt/jdk1.7.0_60/jre/bin/java 2000
echo "/opt/jdk1.7.0_60/jre/lib/arm/jli" | tee -a /etc/ld.so.conf
ldconfig

Runs better inside xserver xsdl make sure it opened first.

type

Apt-get install openbox

Then make runescape script type

echo "env DISPLAY=\$1 /usr/bin/openbox & env DISPLAY=\$1 /opt/runescape/runescape --prmfile=oldschool.prm &" | tee /usr/bin/oldschool
chmod 755 /usr/bin/oldschool

To run it type 

expot DISPLAY=:0 PULSE_SERVER=tcp:127.0.0.1:4712

Then

oldschool :0.0

**Comment by fatchan** [comment]
Nearly perfect, I just had to escape the $ in the script
so replace:

echo "env DISPLAY=$1 /usr/bin/openbox & env DISPLAY=$1 /opt/runescape/runescape --prmfile=oldschool.prm &" | tee /usr/bin/oldschool

WITH

echo "env DISPLAY=\$1 /usr/bin/openbox & env DISPLAY=\$1 /opt/runescape/runescape --prmfile=oldschool.prm &" | tee /usr/bin/oldschool

definitely improves fps. I was getting like 3fps with the app on my galaxy s8 so I know my phone wasnt the issue. Running it this way gets AMAZING fps at least 30 its so smooth crazy!

thanks a ton! Posting a guide on reddit :D

**Comment by codydixon** [comment]
Yes I found that this version of java runs the best :) and I also noticed it even run better on linuxdeploy. Root needed. But it does run better. I have a galaxy note 4 verizon 32 bit snapdragon if you are running a 64 bit octacore I think you should run the soft float version butaye this one works on the 64 bit too. Maybe a test. Try this command. 

wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/7u60-b19/jdk-7u60-linux-arm-vfp-sflt.tar.gz
tar -C /opt -zxvf jdk-7u60-linux-arm-vfp-sflt.tar.gz
rm jdk-7u60-linux-arm-vfp-sflt.tar.gz
update-alternatives --install /usr/bin/java java /opt/jdk1.7.0_60/jre/bin/java 2000
echo "/opt/jdk1.7.0_60/jre/lib/arm/jli" | tee -a /etc/ld.so.conf
ldconfig

Try that. Also. I ask what resolution are yiu running. On linux deploy app using ubuntu I can run a resolution of 1024×576 in fullscreen mode and have around 24 to 35 frams. But in 768×620 I can run nonFullscreen mode and it runs 60 frams it looks like.

**Comment by Kodyhtcuser** [comment]
Hey can I get a little help I am using HTC desire 530. I have installed and run Gnuroot Debian and the runescape add on. It worked just really low fps. I have tried figuring out how to use your fix but I am brand new to this. Do I run Gnuroot runescape let it finish to the osrs menu then go back to the terminal and start entering commands on one window or create a new window with each command? And trying to figure out X server.

**Comment by Kodyhtcuser** [comment]
And I am not rooted if that makes a difference
